### PR TITLE
Use Thrift 0.13.0

### PR DESCRIPTION
**Pull request by Falco20019**
## Which problem is this PR solving?
- https://github.com/jaegertracing/jaeger/issues/1972
- https://github.com/uber/tchannel-go/issues/781
- https://github.com/jaegertracing/jaeger-client-csharp/issues/139
- https://github.com/jaegertracing/jaeger-idl/issues/62

## Which PRs is this blocked on?
- https://github.com/jaegertracing/jaeger-idl/pull/63
- https://github.com/jaegertracing/docker-protobuf/pull/15

## Short description of the changes
- The Go and C# clients need to upgrade Thrift to `0.13`. Go because of a inherited Prometheus dependency, C# because of strong naming assemblies. The current `jaeger-idl` only produces `0.9.2`. Right now, we have to manually adjust the `Makefile` to use the necessary version. There is no way to also use this as submodule.

I would like to check if other libraries need to stay on `0.9.2` for any reason or if we can upgrade this for all. If any have to stay on the old version, we might modify the Makefile to use different versions for the different clients.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-THRIFT_VER=0.9.2
-THRIFT_IMG=thrift:$(THRIFT_VER)
+THRIFT_VER=0.13
+THRIFT_IMG=jaegertracing/thrift:$(THRIFT_VER)
```
Comment (pavolloffay) [inline Makefile:3]: These two should be made configurable to allow submodules modify the build.

```
THRIFT_VER ?= 
```

**Comment by pavolloffay** [comment]
The client libraries can use an arbitrary thrift version. The version set in this repository is used for the CI job. 

We should make the version configurable so anybody pulling the makefile from this repository can override it.

File: Makefile (hunk 1 above)
Comment (Falco20019) [reply Makefile:3]: Currently only the directories are configurable. I currently reworked a lot of the `Makefile` in https://github.com/jaegertracing/jaeger-idl/pull/63 but this still doesn't make it configurable.

We might want to have a separate PR for making all docker versions configurable. I tried to keep the change in this PR minimal.

Comment (pavolloffay) [reply Makefile:3]: >  I tried to keep the change in this PR minimal.

The changes I am asking for are for the lines that are changed in this PR.

I am not asking to change other docker versions just this one. The issue is that the `jaegertracing/thrift` contains only a single version.

Comment (Falco20019) [reply Makefile:3]: I'm fine with it. Adjusted it accordingly :) Might still be a good idea to discuss having all the others also configurable in a separate PR once this is in.

**Comment by pavolloffay** [comment]
There is a conflict in the Makefile

**Comment by pavolloffay** [comment]
thanks @Falco20019

**Comment by Falco20019** [comment]
Thank you! :)

**Pull request merged**